UKHomeOffice/sgmr-service
# NMSW-34 Add detail/summary component back in

## AC

Match the styling of the existing details component

----

## To test

- open a voyage departure or arrival page
- there should be the > for other port question
- the arrow should spin on the open/close of the question

----

## Notes

On investigation I decided to keep the detail/summary component from GDS as it meets the styling required and afaik has been tested for accessibility on the GDS side.
